lsp/ty.lua

```diff
@@ -0,0 +1,24 @@
+---@brief
+---
+--- https://github.com/astral-sh/ty
+---
+--- `ty`, a static type checker and language server for python.
```

Comment:
It's just ty (no backticks) unless it references the executable `ty`.

```suggestion
--- A Language Server Protocol implementation for ty, an extremely fast Python type checker and language server, written in Rust.
```